api/schema/application_aws_resources_resource.go

```diff
@@ -0,0 +1,185 @@
+					ElementType: types.StringType,
+					Description: "IDs of AWS direct connect connections to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_directconnect_virtualinterfaces": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS direct connect virtual interfaces to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_customergateways": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 customer gateways to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_egressonlyinternetgateways": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 egress only internet gateways to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_eips": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 eips to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_flowlogs": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 flow logs to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_instances": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 instances to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_instanceconnectendpoints": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 instance connect endpoints to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_internetgateways": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 internet gateways to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_natgateways": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 nat gateways to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_networkacls": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 network acls to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_networkinterfaces": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 network interfaces to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_routetabless": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 route tabless to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_securitygroups": resource_schema.SetAttribute{
```

Comment:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group
```suggestion
				"aws_security_group_ids": resource_schema.ListAttribute{
```